src/ui/server/helpers/summary-lists/policy-and-export/single-contract-policy-fields/index.ts

```diff
@@ -0,0 +1,39 @@
+import { FIELDS } from '../../../../content-strings/fields/insurance';
+import { FIELD_IDS } from '../../../../constants';
+import fieldGroupItem from '../../generate-field-group-item';
+import formatDate from '../../../date/format-date';
+import formatCurrency from '../../../format-currency';
+import { ApplicationPolicyAndExport, SummaryListItemData } from '../../../../../types';
+
+const {
+  INSURANCE: {
+    POLICY_AND_EXPORTS: {
+      CONTRACT_POLICY: {
+        SINGLE: { CONTRACT_COMPLETION_DATE, TOTAL_CONTRACT_VALUE },
+      },
+    },
+  },
+} = FIELD_IDS;
```

Comment:
like above, can we do FIELD_IDS.INSURANCE.POLICY_AND_EXPORTS

Reply:
Good shout i've done the same as above - importing directly from `policy-and-exports` `FIELD_IDS`, instead of the root.